docs/faq.md

```diff
@@ -20,15 +20,42 @@ Skin can be simulated using the [AdnSkin](maya/skin) deformer. This deformer req
 
 Similarly to a skin simulation setup, the fascia can be simulated using [AdnSkin](maya/skin) deformer as well. In this case, it is recommended to use values of *Rest Length Multiplier* lower than 1. This deformer also requires one mesh (a single target, i.e. the muscle geometries combined) or multiple meshes (multiple targets, i.e. the muscle geometries separated) to drive the simulation and a mesh to apply the deformer to (fascia geometry).
 
-## Can I simulate muscles, fascia and skin all coupled?
+## How can I generate the fascia geometry?
 
-Yes, you can simulate muscles, fascia and skin following these steps:
+In order to obtain a plausible fascia geometry, we can use two techniques that require the muscle geometries and the skin geometry as inputs.
+
+The easiest option is to use the shrinkwrap deformer provided by Maya. This solution works better for low poly geometries.
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the combined geometry of all muscles, then select the copy of the skin geometry.
+- Apply the shrinkwrap deformer and adjust its settings until the desired fascia geometry is achieved.
+- Delete the history of the fascia geometry.
+
+For high-end fascia simulations, a better option would be to create the fascia geometry using a pressured cloth simulation. nCloth can be used to achieve this:
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the fascia geometry and click nCloth > Create nCloth.
+- Select the muscle geometries and click on nCloth > Create Passive Collider.
```

Comment:
I think you should combine first, not sure. Can you please double check? If combining is needed, please add a step before.

Reply:
Correct! Also, in my setups I also included the bones in some cases to avoid more intersections. I would add that.
And optionally I also tweaking the friction and stickiness parameters on the nCloth to avoid the cloth simulation to slide too much over the muscles and fold. I let you decide if we add that or not @carlosmonteagudoinbibo

Reply:
Added the step for combining the muscles. Let me know if we should add more details like including bones or changing other parameters in nCloth. @carlosmonteagudoinbibo

Reply:
Good point again, let's double check with Francesco and let's follow his suggestion.

Reply:
CC: @francescogioiainbibo